### ISSUE TITLE
MODX 3: Buttons - different button groups

### ISSUE THREAD

**Issue by jonleverrier** [modxcms/revolution#14006]
### Summary
We have a great new button pattern when editing resources;

![screen shot 2018-07-19 at 21 15 06](https://user-images.githubusercontent.com/2373940/42964975-e7d8513a-8b98-11e8-8af5-198638651858.png)

### Observed behavior
I wonder if this should be applied to all button groups to make it consistent?

**button group when editing template, tv, chunk, snippet, plugin:**
/manager/?a=element/template/update&id=1
/manager/?a=element/tv/update&id=1
/manager/?a=element/chunk/update&id=1
/manager/?a=element/snippet/update&id=1
/manager/?a=element/plugin/update&id=1
![screen shot 2018-07-19 at 21 16 51](https://user-images.githubusercontent.com/2373940/42965053-2a1224fe-8b99-11e8-83c7-f9f5da1fc5b1.png)

**button group when updating a user:**
/manager/?a=security/user/update&id=1
![screen shot 2018-07-19 at 21 20 09](https://user-images.githubusercontent.com/2373940/42965184-92a97120-8b99-11e8-9974-c878f1482afa.png)

**button group when editing a form customisation set:**
/manager/?a=security/user/update&id=1
![screen shot 2018-07-19 at 21 20 50](https://user-images.githubusercontent.com/2373940/42965232-ad2bc8a4-8b99-11e8-915d-9eda1f2a87ac.png)

**button group when updating a dashboard group:**
/manager/?a=system/dashboards/update&id=1
![screen shot 2018-07-19 at 21 20 50](https://user-images.githubusercontent.com/2373940/42965282-cf1d5cb6-8b99-11e8-96c1-374083e89508.png)

**button group when updating a context**
/manager/?a=context/update&key=web
![screen shot 2018-07-19 at 21 36 59](https://user-images.githubusercontent.com/2373940/42965995-fa8e950c-8b9b-11e8-80c3-7748c3cdc001.png)

**button group when updating a user group**
/manager/?a=security/usergroup/update&id=1
![screen shot 2018-07-19 at 21 38 51](https://user-images.githubusercontent.com/2373940/42966066-353ca3ba-8b9c-11e8-9171-3c1e6cbf4df8.png)

**button group when updating an access policy / policy template**
/manager/?a=security/access/policy/update&id=2
/manager/?a=security/access/policy/template/update&id=1
![screen shot 2018-07-19 at 21 38 51](https://user-images.githubusercontent.com/2373940/42966112-6a44918a-8b9c-11e8-84bb-52a58b4d4acd.png)

**button group when updating a media source**
/manager/?a=source/update&id=1
![screen shot 2018-07-19 at 21 43 20](https://user-images.githubusercontent.com/2373940/42966235-cfd6272a-8b9c-11e8-8b3f-b0695f0b67ee.png)

### Expected behavior
To follow the new button pattern that the resource page uses.

I appreciate that the resource page is the most import for content editors and the other parts of the CMS are more for site builders, but surely 1 pattern is better than 2?

### Environment
3.x branch built from github

**Comment by SnowCreative** [comment]
Personally, I don't think "Close" should ever be in a group; it's used much too often by users in my experience.

**Comment by JoshuaLuckers** [comment]
It's also weird that we are hiding functionality behind a dropdown menu that requires clicking.

**Comment by Ruslan-Aleev** [comment]
Yes, I also do not understand why hide the "Copy" button for example?

Why not to leave the same kind of buttons in MODX3 ?

![42965053-2a1224fe-8b99-11e8-83c7-f9f5da1fc5b1 1](https://user-images.githubusercontent.com/12523676/65340643-bc02c980-dbdf-11e9-9ded-c398b1e38510.png)

**Comment by rthrash** [comment]
It's very pretty and clean, but I'm not sure quite as good on the UX front. 

I wonder how it would work to change the help button into a clickable "?" icon and leaving the remaining buttons as text-only? I might could buy into the checkmark icon on the primary action (save), but adding icons to all the others gets too busy for my personal preference.

**Comment by Ruslan-Aleev** [comment]
Yes, the text + icons, it seems to me, looks superfluous.
We can think of something with various options, for example:

![buttons_1](https://user-images.githubusercontent.com/12523676/65345406-43096f00-dbeb-11e9-9061-8a6912437358.png)

![buttons_3](https://user-images.githubusercontent.com/12523676/65345408-43096f00-dbeb-11e9-8016-d3d3f70ef123.png)

![buttons_2](https://user-images.githubusercontent.com/12523676/65345407-43096f00-dbeb-11e9-9ad5-ba31f0553e98.png)

![buttons_4](https://user-images.githubusercontent.com/12523676/65345661-d5117780-dbeb-11e9-8cba-24310e05bf68.png)

The help button in the form of an icon will look appropriate, because this button is not often needed :)
By the way, the delete button can also be made an icon - there will be less false clicks. If you need to remove - you need to aim :)
